web/templates/modules/module_configuration.html

```diff
@@ -19,7 +19,24 @@ <h4 class="title">{{data.module.name}}</h4>
                     {% endif %}
 
                     <form class="form-horizontal" method="post" action="{{url_for('ModulesView:configure', id=data.module._id)}}">
-                    {% if data.module.type == "Processing" %}
+                    {% if data.module.type == "Preloading" %}
+                        <hr />
+
+                        <div class="form-group">
+                            <label for="acts_on" class="col-sm-2 control-label">Acts On</label>
+                            <div class="col-sm-8">
+                                <input type="text" class="form-control" name="acts_on" value="{{data.module.acts_on|smart_join}}">
+                                <p class="help-block">Comma-delimited list of types that this module is able to handle (ex: word,excel).</p>
+                            </div>
+                        </div>
```

Comment:
I don't think Preloading Modules have an acts_on

Reply:
Slipped through while simplifying the preloading module implementation. Will remove it.